pkg/state/state.go

```diff
@@ -382,13 +382,12 @@ func (st *HelmState) SyncRepos(helm RepoUpdater, shouldSkip map[string]bool) ([]
 		if shouldSkip[repo.Name] {
 			continue
 		}
-		var err error
+		var err error = nil
```

Comment:
```suggestion
		var err error
```

AFAIK the zero value for `error` is nil so we can just write it like this 👍